deployable/java/Dockerfile.cloudfunctions

```diff
@@ -0,0 +1,55 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# compile local java-logging library
+FROM maven:3.8.4 AS lib-env
+WORKDIR /app
+COPY _library ./
+RUN mvn verify
+RUN mvn -Dmaven.test.skip=true package
+# copy the latest jar file to java-logging.jar
+RUN cp `ls google-cloud-logging/target/*.jar | grep -v "test" | sort -V | tail -n1` /app/java-logging.jar
+RUN cp `ls proto-google-cloud-logging-v2/target/*.jar | grep -v "test" | sort -V | tail -n1` /app/proto-java-logging.jar
```

Comment:
nit: consider combining multiple RUN commands to reduce the size of the image, hence improve building time. Same could be apply to the next image to improve uploading and launching times.